### PR TITLE
Milestone 1 branch

### PR DESCRIPTION
**Pull request by daviidy**
We created 2 main directories:

- lib - for the game logic

- bin - with the main.rb file

We made our main.rb file displaying “Hello world!” when we use bin/main.rb as a bash command.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env ruby
```
Comment (moinkhanif) [inline bin/main.rb:1]: - [x] **[REQUIRED]** As pointed out by the linter, and also the project specifications, the `main.rb` file doesn't have executable permissions. Test if executable file works (bin/main.rb displays message as well as ruby bin/main.rb) and fix this, please.
Reference project specification: 

-  [x] **[REQUIRED]** After implementing the above change, please fix any linter errors before project submission.

**Review by moinkhanif — CHANGES_REQUESTED** [review]
## Changes Required ❗

Hey team, nice work implementing Milestone 1 so far. However, please **fix** the requested changes in the **comments below** to move forward with this project. 👍

If you have any questions, you can reach out to me on slack at @moinkhanif.
**Happy Coding!**

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
#### Hello Team 👋
#### Status: Changes Required ♻️

Great job so far! 👏👋
Let's cover a few details together to make your project better 😊⏬

### Read the Documents Please 😇

💡 This will be a kind of repetition of the last review. 😇
After you read the documentation, please let me know if you need anything more, through Slack.

- https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/

- [ ] [REQUIRED] `main.rb` is an [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/):
  - `#!/usr/bin/env ruby` is the top line (you shouldn't delete it)
  - puts "Hello world" in the following line
    ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- [ ] [REQUIRED] Executable file should work with these commands:
 - `bin/main.rb`
 - `ruby bin/main.rb`)
 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

 📸 Screenshot from the project

image

 📸 Screenshot from the file properties

image

## That's it, you can request another code review request after making the changes ♻️😇

With all my respect 🙏
> Kubilay Caglayan | Technical Support Engineer

**Comment by mimipeshy** [comment]
### STATUS : APPROVED !! :partying_face:  :dancers: 
Congratulations in making the required changes. This milestone is approved !!  :clap:  :clap: Good luck in the next project !!

Happy Coding :computer: 

Peris Ndanu

**Pull request merged**